functions-methods.js

```diff
@@ -33,4 +71,30 @@
 // checkEmailValidity("tessmellink@novi.nl") geeft true - want @ en punt op de juiste plek
 // checkEmailValidity("n.eekenanovi.nl") geeft false - want geen @
 // checkEmailValidity("n.eeken@novinl.") geeft false - want de punt mag niet als laatst
-// checkEmailValidity("tessmellink@novi,nl") geeft false - want er staat een komma in
\ No newline at end of file
+// checkEmailValidity("tessmellink@novi,nl") geeft false - want er staat een komma in
+
+function checkEmailValidity(email ){
+
+    // includes @
+    const includesAt = email.includes('@');
+//    geen , komma
+    const includesComma = email.includes(',');
+// géén punt    als laatste
+    const lastDot = email.lastIndexOf('.')
+    const containsNoDotAtEnd = lastDot !== email.length - 1;
+
+    if (includesAt && !includesComma && containsNoDotAtEnd){
+        return true;
+
+    } else {
+        return false;
+    }
```

Comment:
Je zou meteen de uitkomst terug kunnen geven
```suggestion
    return includesAt && !includesComma && containsNoDotAtEnd
```